src/page/Home/hooks/useThemes.ts

```diff
@@ -1,14 +1,14 @@
 import { requests } from '@/api/requests';
 import type { ThemeInfo } from '@/types';
-import { useQuery } from '@tanstack/react-query';
+import { useSuspenseQuery } from '@tanstack/react-query';
 
 const useThemes = () => {
-  const { data, error, isLoading } = useQuery<ThemeInfo[]>({
+  const { data } = useSuspenseQuery<ThemeInfo[]>({
     queryKey: ['themeDatas'],
     queryFn: requests.fetchTheme,
   });
 
-  return { themes: data, isLoading, error };
+  return { themes: data };
```

Comment:
마찬가지로 themes라고 이름을 지정하는 방법도 있고, 그냥 밖에서 편하게 사용하도록 data 자체를 바로 return 하는 방법도 있어요!

```suggestion
  return data;
```